packages/frontend/src/components/ReactHookForm/MultiKeyValuePairsInput.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, FormGroup } from '@blueprintjs/core';
+import { useFieldArray, useFormContext } from 'react-hook-form';
+import Input from './Input';
+import {
+    AddKeyValuePairRow,
+    MultiKeyValuePairRow,
+} from './MultiKeyValuePairsInput.styles';
+
+type Props = {
+    name: string;
+    label: string;
+};
+export const MultiKeyValuePairsInput = ({ name, label }: Props) => {
+    const {
+        control,
+        register,
+        formState: { errors },
+    } = useFormContext();
+    const { fields, remove, append } = useFieldArray({ name, control });
+    return (
+        <FormGroup label={label}>
```

Comment:
We should use the `<InputWrapper >` component here so it gets the same label styling and logic as the other form input components

Reply:
Is the logic in `InputWrapper` compatible with the custom form control I have by using `fieldArray`? Since this would be wrapping all the Inputs rather than just a single input, if that makes sense?

Reply:
I could just take the `className="input-wrapper"` for styling and leave the custom logic for handling the multi-input form?